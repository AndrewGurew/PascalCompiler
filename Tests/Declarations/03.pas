var
a,b:integer;
c:double;