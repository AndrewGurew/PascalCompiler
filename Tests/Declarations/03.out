Declaration
 ⎬b - Integer(Var)
 ⎬a - Integer(Var)
 ⎬c - Double(Var)